src/Views/scstormrunoff.html

```diff
@@ -134,10 +134,131 @@ <h4>Citation</h4>
+                                    </div>
+                                    <div class="form-column">
+                                        <label for="StandardCurveNumber">Standard Curve Number, CN</label> <span ng-if="SyntheticUrbanHydrograph.StandardCurveNumber.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                    <input type="number" name="StandardCurveNumber" ng-model="vm.standardCurveNumber" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                </span>
+                                <span class="form-row">
+                                    <div class="form-column">
+                                        <label for="WatershedRetention">Watershed Retention, S</label> <span ng-if="SyntheticUrbanHydrograph.WatershedRetention.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="WatershedRetention" ng-model="vm.watershedRetention" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                    <div class="form-column">
+                                        <label for="InitialAbstraction">Initial Abstraction</label> <span ng-if="SyntheticUrbanHydrograph.InitialAbstraction.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                        <input type="number" name="InitialAbstraction" ng-model="vm.initialAbstraction" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                    </div>
+                                </span>
+
+                                <label for="TimeOfConcentrationMin">Time of Concentration, Tc (minutes)</label> <span ng-if="SyntheticUrbanHydrograph.TimeOfConcentrationMin.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than or equal to 0</span>
+                                <input type="number" name="TimeOfConcentrationMin" ng-model="vm.timeOfConcentrationMin" ng-minlength="0" ng-pattern="greaterThanOrEqualToZero" required/>
+                                <label for="TimeOfConcentration">Select Time of Concentration</label>
+                                <select name="TimeOfConcentration" ng-model="vm.SelectedTimeOfConcentration" ng-options="time.name for time in vm.TimeOfConcentrationOptions track by time.value"></select>
+                                
+                                <!-- if Lag Time Equation is selected -->
```

Comment:
Add a title to each Time of Concentration form section, so that the user knows these form elements are related to the Time of Concentration calculation method. Something simple like "Lag Time Equation" and "Travel Time Method" in bold is fine.

Reply:
Should the title be above the "Time of Concentration, Tc (minutes)" question? (which is right above the drop down to select Lag Time or Travel Time) Or should the title be above the dynamic form questions that generate, like in this picture?
![timecontlabel](https://user-images.githubusercontent.com/76274215/182416505-25be0ed7-838b-4600-81e3-72f77b3bbab6.PNG)

Reply:
The title should be above the dynamic form questions 👍